Exercises/NoModules/Chapter 09/Soln9_02.cpp

```diff
@@ -0,0 +1,55 @@
+// Exercise 9-2 Working with std::string_view<> and std::span<>
+#include <iostream>
+#include <format>
+#include <string_view>
+#include <span>
+
+// The function prototype including defaults for parameters
+void show_data(
+  std::span<const int> data,
```

Comment:
My text says:
"Exercise 9-2. Adjust Ex8_11 to use std::string_view"
but you are converting 8_11 to using string_view and span.
Did you update the exercise in the text?

Reply:
I did just that: "Adjust Ex8_11 to use std::string_view and std::span<>."